pyoos/parsers/ioos/describe_sensor.py

```diff
@@ -8,7 +10,7 @@
 class IoosDescribeSensor(object):
 
     def __new__(cls, element):
-        if isinstance(element, str):
+        if isinstance(element, bytes):
```

Comment:
On second thought `six.string_types` will be `str` on Python 3 and we do need `bytes` to avoid issues in the pipeline. I confess I am terrible defining a good strategy to handle `text` types conversions. I am never clear on the best way to do it...